build.gradle

```diff
@@ -34,7 +34,7 @@ repositories {
 ext {
     opensaml_version = '3.4.3'
     dropwizard_version = '1.3.16'
-    utils_version = '2.0.0-371'
+    utils_version = '2.0.0-375'
```

Comment:
How do you know to bump to latest versions? I wouldn't have thought to do this.

Reply:
Ah, I had to update the rest-utils library in this PR to allow a GET request with added headers. This was to allow the client to make a request and pass Istio headers along like we do elsewhere. https://github.com/alphagov/verify-utils-libs/pull/49